File: app/src/main/java/br/com/erico/desafio_android/models/RepositoryResponse.java
package br.com.erico.desafio_android.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class RepositoryResponse {

    @SerializedName("items")
    @Expose
    private List<Repository> repositories = null;

    @SerializedName("total_count")
    @Expose
    private Integer numRepositories;

    public List<Repository> getRepositories() {
        return repositories;
    }

    public void setRepositories(List<Repository> repositories) {
        this.repositories = repositories;
    }

    public Integer getNumRepositories() {
        return numRepositories;
    }

    public void setNumRepositories(Integer numRepositories) {
        this.numRepositories = numRepositories;
    }
}

File: app/src/main/java/br/com/cavreti/desafio_android/network/response/RepositoryResponse.kt
package br.com.cavreti.desafio_android.network.response

import br.com.cavreti.desafio_android.data.Repository
import com.google.gson.annotations.SerializedName

data class RepositoryResponse(
    @SerializedName("total_count") var totalCount: Int,
    var items: List<Repository>)